lizrd/support/sync_code.py

```diff
@@ -80,14 +80,15 @@ def athena_user_to_workdir(connection):
         "plgmaciejpioro": "maciejpioro",
         "plgludziej": "ludziej",
         "plgsimontwice": "simontwice",
+        "plgkciebiera": "plgkciebiera",
```

Comment:
athena_user_to_workdir is not used anymore

Reply:
Yeeted